src/main/java/strategy/OperationStrategyImpl.java

```diff
@@ -0,0 +1,25 @@
+package strategy;
+
+import java.util.Map;
+import model.FruitTransaction;
+import strategy.handler.BalanceOperationHandler;
+import strategy.handler.PurchaseOperationHandler;
+import strategy.handler.ReturnOperationHandler;
+import strategy.handler.SupplyOperationHandler;
+
+public class OperationStrategyImpl implements OperationStrategy {
+    private static final Map<FruitTransaction.Operation, OperationHandler> operationHalndlerMap;
+
+    static {
+        operationHalndlerMap = Map.of(
+                FruitTransaction.Operation.BALANCE, new BalanceOperationHandler(),
+                FruitTransaction.Operation.SUPPLY, new SupplyOperationHandler(),
+                FruitTransaction.Operation.PURCHASE, new PurchaseOperationHandler(),
+                FruitTransaction.Operation.RETURN, new ReturnOperationHandler());
+    }
```

Comment:
You should have like:
```suggestion
public class OperationStrategyImpl implements OperationStrategy {
    private final Map<FruitTransaction.Operation, OperationHandler> operationHalndlerMap;

    public OperationStrategyImpl(Map<FruitTransaction.Operation, OperationHandler> operationHalndlerMap) {
        this.operationHalndlerMap = operationHalndlerMap;
    }
```

Reply:
Firstly your comment was "It is better to use constructor". And there was not obligation make this changes. And now your comment looks the same. In both cases I don;t understand where I have to initialize my Map with defined set of operations. That's why my logic was to do like this and therefore I always can add/remove certain operation if it's needed. Explain please